CMakeLists.txt

```diff
@@ -54,32 +59,42 @@ set(WOLFSSL_LIBRARY wolfssl)
 
 add_subdirectory(extern/curl)
 
+# uWebSockets setup
+set(WITH_WOLFSSL ON)
+set(WITH_LIBUV ON)
+# Set the correct libuv library name for uWebSockets
+set(LIBUV_LIBRARY uv_a)
+include(extern/uSockets.cmake)
+include(extern/uWebSockets.cmake)
+
+# Define include directories for uWebSockets and uSockets
+set(UWEBSOCKETS_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/extern/uWebSockets/src)
+set(USOCKETS_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/extern/uWebSockets/uSockets/src)
+
 #
 
 add_executable(Lute.CLI)
 set_target_properties(Lute.CLI PROPERTIES OUTPUT_NAME lute)
 
-set(LIBUV_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/extern/libuv/include)
-
 include(Sources.cmake)
 
 target_compile_features(Lute.Runtime PUBLIC cxx_std_17)
 target_compile_features(Lute.Fs PUBLIC cxx_std_17)
 target_compile_features(Lute.Luau PUBLIC cxx_std_17)
-target_compile_features(Lute.Net PUBLIC cxx_std_17)
+target_compile_features(Lute.Net PUBLIC cxx_std_20)
```

Comment:
Wait, does uWebSockets require C++20?

Reply:
yes, but I've moved this requirement to uWebSockets.cmak, which is better, hopefully 😅